application/src/main/resources/extensions/notification.yaml

```diff
@@ -59,6 +59,10 @@ spec:
               value: "TLS"
             - label: "不加密"
               value: "NONE"
+        - $formkit: text
```

Comment:
It might be better to place this setting option under the username option, and it is recommended to add a help attribute, for example: `如果用户名为实际发信地址，可忽略`


<img width="668" alt="image" src="https://github.com/halo-dev/halo/assets/21301288/511aa693-69ab-451a-839d-7f878cea2fb1">